docs/src/template.html

```diff
@@ -8,6 +8,8 @@
     %sapper.base%
 
     <link rel="stylesheet" href="global.css" />
+    <link rel="stylesheet" href="docs.min.css" />
+    <link rel="stylesheet" href="svant.min.css" />
     <link href="prism/prism.css" rel="stylesheet" />
```

Comment:
I just noticed that the code style has changed just load `prism.css` before the `docs.min.css` and it fixes it